google-cloud-graalvm-samples/basic/pubsub-sample/pom.xml

```diff
@@ -4,7 +4,7 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
   <parent>
-    <artifactId>google-cloud-graalvm-samples</artifactId>
+    <artifactId>basic</artifactId>
```

Comment:
Maybe `com.google.cloud:google-cloud-graalvm-samples-basic`? Otherwise, the artifact is not sufficiently scoped, unless you start changing the `groupId`.